public/static/plugins/perf/js/trend_chart.js

```diff
@@ -122,7 +122,7 @@ mciModule.factory('PerfChartService', function() {
   // thread `level` (ignored in this case) and series `item`
   function getValueForMaxOnly() {
     return function(item) {
-      return item[cfg.valueAttr]
+      return item.hasOwnProperty(cfg.valueAttr) ? item[cfg.valueAttr] : null
```

Comment:
I don't think this change required in this case.

Reply:
You are correct, this one is not needed.